elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -486,7 +446,7 @@ public User accessUser(Object opaqueUser) {
     }
 
     @Override
-    public void setRequestScope(RequestScope requestScope) {
+    public void setRequestScope(com.yahoo.elide.core.RequestScope requestScope) {
```

Comment:
Another question - why does this method exist given that every method takes request scope?  Can we just remove this method?